index.html

```diff
@@ -1613,11 +1617,13 @@ <h3 id="h-track-buffers" resource="#h-track-buffers"><span property="xhv:role" r
+          These presentation time ranges are based on <a href="#presentation-timestamp">presentation timestamps</a>, frame durations, and potentially coded frame group start times for coded frame groups across track buffers in a muxed <a href="#idl-def-SourceBuffer" class="idlType"><code>SourceBuffer</code></a>.
+          For specification purposes, this information is treated as if it were stored in a <a href="https://www.w3.org/TR/html5/embedded-content-0.html#normalized-timeranges-object">normalized TimeRanges object</a>. Intersected <a href="#track-buffer-ranges">track buffer ranges</a> are used to report <code><a href="https://www.w3.org/TR/html5/embedded-content-0.html#dom-media-buffered">HTMLMediaElement.buffered</a></code>, and must therefore support uninterrupted playback within each range of <code><a href="https://www.w3.org/TR/html5/embedded-content-0.html#dom-media-buffered">HTMLMediaElement.buffered</a></code>.
+
+          </p><div class="note"><div class="note-title marker" aria-level="4" role="heading" id="h-note18"><span>Note</span></div><p class="">These coded frame group start times differ slightly from those mentioned in the <a href="#sourcebuffer-coded-frame-processing">coded frame processing algorithm</a> in that they are the earliest <a href="#presentation-timestamp">presentation timestamp</a> across all track buffers following a discontinuity, which can occur within the <a href="#sourcebuffer-coded-frame-processing">coded frame processing algorithm</a> or result from the <a href="#sourcebuffer-coded-frame-removal">coded frame removal algorithm</a>, regardless of <code><a href="#widl-SourceBuffer-mode">mode</a></code>.
+          The threshold for determining disjointness of <a href="#track-buffer-ranges">track buffer ranges</a> is implementation-specific.
+          To approximate the discontinuity detection logic within the <a href="#sourcebuffer-coded-frame-processing">coded frame processing algorithm</a>, it is recommended when determining if one time range is continuous with another as a result of newly processed or removed coded frames, to allow continuity across no more than twice the maximum frame duration seen so far by this <a href="#track-buffer">track buffer</a> between the earlier range's end timestamp and the later range's earliest <a href="#presentation-timestamp">presentation timestamp</a>. In a muxed <a href="#idl-def-SourceBuffer" class="idlType"><code>SourceBuffer</code></a>, implementations may choose to use the earliest <a href="#presentation-timestamp">presentation timestamp</a> across all <a href="#track-buffer">track buffers</a> following a discontinuity detected by <a href="#sourcebuffer-coded-frame-processing">coded frame processing algorithm</a> as the range start for the track buffers across that muxed group of frames.</p></div>
```

Comment:
Note, Chrome is just a little more strict than this. If max frame duration seen so far is "10", Chrome would find the following to be discontinuous (notation: {coded frame PTS, duration}, assuming everything is a keyframe for simplicity in these examples):
{0,10} and {21,10} would be discontinuous ranges [0,10) [21,31)
{0,10} and {19,10} would be continuous [0,29)
This is because Chrome operates on the delta (more like the coded frame processing algorithm) between the earlier range's highest PTS (not end timestamp) and the next range's earliest PTS being within 2 * max duration seen so far.  (Note that Chrome uses DTS incorrectly still, but ignore that for the purposes of this discussion.) There are still some hairy edge cases that don't conform to even this regime in Chrome (like the frame duration used in calculating the max seen so far isn't precisely that in all cases), hence the "implementation-specific" and non-normative note framing of this.